hi-ml/testhiml/testhiml/test_box_utils.py

```diff
@@ -8,27 +8,78 @@
 from health_ml.utils.box_utils import Box, get_bounding_box
```

Comment:
Same, please add the header with copyright info

Reply:
I think this file already has it.